tests/data/placebo/test_ec2_userdata/ec2.py

```diff
@@ -0,0 +1,1638 @@
+# Copyright 2015-2017 Capital One Services, LLC
```

Comment:
this file shouldn't be here, it looks like it was copied over accidentally and can be removed.

Reply:
oops! Good catch, thanks. Fixed.